Command/CreateDatabaseDoctrineCommand.php

```diff
@@ -79,10 +79,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $error = false;
         try {
-            $shouldNotCreateDatabase = $ifNotExists && in_array($name, $tmpConnection->getSchemaManager()->listDatabases());
+            $cleanName = str_replace('`', '', $name);
```

Comment:
Also quoting the identifier [here](https://github.com/ggirtsou/DoctrineBundle/blob/master/Command/CreateDatabaseDoctrineCommand.php#L77) does not always enclose the database name in backticks. This is only working on MySQL but not for other platforms!